### ISSUE TITLE
503 message should obey language

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/server#20812]
@MorrisJobke as we discussed

```
curl --request PROPFIND \
  --url http://localhost/nc/remote.php/webdav/Nextcloud/ \
  --header 'accept-language: de-DE' \
  --header 'authorization: Basic dG9iaTpTSGZ6ZC04YkQyNC1xOUtSaS1zU2NRay03ZHh4eA==' \
  --header 'content-type: application/xml' \
  --cookie 'ocjycgrudn78=0v34jur3gq15b9520kgp2neuun; oc_sessionPassphrase=LKB5YRfQzsgEAI0wdgEtQUKdwnZzlisz3onYBIBDvs6ukoDZLJ%252FlbofTcseKYG9qy4w2kUAqD3C148xFmPtBv52rjUZtnIaITzvXH%252FL03dYwweKON5hVhtUhEjV11%252BEv; nc_sameSiteCookielax=true; nc_sameSiteCookiestrict=true; PHPSESSID=1qa3tqfh252ksha1uhmoa4a76d' \
  --data '<?xml version="1.0" encoding="UTF-8"?>
<D:propfind xmlns:D="DAV:">
<D:prop>
<D:displayname/>
<D:resourcetype/>
<favorite xmlns="http://owncloud.org/ns"/>
<is-encrypted xmlns="http://nextcloud.org/ns"/>
<mount-type xmlns="http://nextcloud.org/ns"/>
<D:creationdate/>
<id xmlns="http://owncloud.org/ns"/>
<D:getetag/>
<permissions xmlns="http://owncloud.org/ns"/>
<share-types xmlns="http://owncloud.org/ns"/>
<share-expiration xmlns="http://nextcloud.org/ns"/>
<rich-workspace xmlns="http://nextcloud.org/ns"/>
<owner-id xmlns="http://owncloud.org/ns"/>
	<X-Hash-MD5 xmlns="http://nextcloud.org/ns"/>
	<D:quota-used-bytes/>
<D:getcontenttype/>
<sharees  xmlns="http://nextcloud.org/ns"/>
</D:prop></D:propfind>'
```

1. have a cal with language de-DE
2. expect to have error message translated

This is useful to have a translated message on clients.
Please make also sure that this works on other error/warning messages <3

**Comment by MorrisJobke** [comment]
Response currently is:

```xml
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>Sabre\DAV\Exception\ServiceUnavailable</s:exception>
  <s:message>System in maintenance mode.</s:message>
</d:error>
```

**Comment by MorrisJobke** [comment]
Fix is in #20844

**Comment by tobiasKaminsky** [comment]
Ref: https://github.com/nextcloud/android/issues/5981
